### PR TITLE
site-recorder|Issue-17-18|Add resolution device & network options

### PR DESCRIPTION
**Pull request by ErezPaperny**
Please review .....
10x

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ Options:
 | -v,--version | output the version number |
 | -d,--debug | see full error messages, mostly for debugging |
 | -gif,--generate-gif | generate gif as additional output |
+| -W, --resolution-width [width] | define the resolution width in the screen recording |
+| -H, --resolution-height [height] | define the resolution height in the screen recording |
+| -D, --device [device] | define the device that will run the screen recording (Override resolution param |
+| -N, --device [network] | define the throttler that will emulate network slowdown |
```
Comment (sidoruk-sv) [inline README.md]:
```suggestion
| -N, --network [network] | define the throttler that will emulate network slowdown |
```

File: README.md [hunk 2]
```diff
@@ -72,6 +76,11 @@ npm install
 npm run example:gif # generates both gif and video
 ```
 
+### Installing the tool as globally
+```sh
+npm install -g .
+```
```
Comment (sidoruk-sv) [inline README.md]: do you need that?
I mean "to checkout repo, and after that installing it globally"

You could install it globally by usual:
```
npm i site-recorder -g
```

or using it without installation by
```
npx site-recorder ...
```

Comment (ErezPaperny) [reply README.md]: You right and I fix documentation. I think its better not to do the npx each time as its running all preRun activities on each run. So I update that install will include the global

**Comment by sidoruk-sv** [comment]
@ErezPaperny could you please merge `master` **into** this branch (to make Travis check work)

**Pull request merged**